src/pages/single-post.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import './single-post.css';
+
+export default class SinglePost extends React.Component<object> {
+
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            error: null,
+            isLoaded: false,
+            posts: [],
+            slug: props.match.params.slug
+        };
+    }
+
+    componentDidMount() {
+        const { slug }: any = this.state;
+        fetch("https://admin.cuhacking.com/ghost/api/v3/content/posts/slug/" + slug + "?key=4ff22bcd0f6effc7d2df8ee82b&include=tags,authors")
```

Comment:
We should store the API information in a JSON, then import it. It should look like this:
```json
{
  "url": "https://admin.cuhacking.com",
  "key": "22444f78447824223cefc48062"
}
```